AF-2748: Heatmap Component: Community Documentation (#3100)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/BAM/BusinessDashboards-section.adoc

This is the main reason why the Dashbuilder project has been included as a core module of the jBPM ecosystem.
Notice also that Dashbuilder, as an independent project, is not only used by jBPM but also by many other projects like, for example, JBoss Teiid a data virtualization system that allows applications to use data from multiple, heterogeneous data stores.

> The _Content Manager_ page gives users access to a rich environment where it is possible to author new content as well as configuring which entries are displayed in the top menu bar. _Content Manager_  can be acessed using the menu *pages*.

Reviewer: ```suggestion
The _Content Manager_ page gives you access to a rich environment where it is possible to author new content as well as configuring which entries are displayed in the top menu bar. You can access _Content Manager_  using the menu *Pages*.
```

.Content Manager Page
image::BAM/ContentManagementPerspective.png[]

[NOTE]
====
By default, {CENTRAL} uses a C3 Renderer API for chart libraries and no longer uses the Google Charts library. To revert to Google Charts,  build it from sources and add it to {CENTRAL} (see the `README.md` file in `dashbuilder-renderer-google`), and then set the renderer system property to `org.dashbuilder.renderer.default=gwtcharts`.
====

== Authoring Pages

> In _Content Manager_  users can create pages which will be composed of components organized in a row and column layout. Components can be dragged to the page editor and a pop-up will be displayed with the component settings that can be modified by the user.

Reviewer: ```suggestion
In _Content Manager_  users can create pages which are composed of components organized in a row and column layout. Components can be dragged to the page editor and a pop-up will be displayed with the component settings that can be modified by the user.
```

.HTML Component Editor
image::BAM/HTMLComponent.png[]

== Reporting Components

.Reporting Components
image::BAM/ReportingComponents.png[]

> Reporting components can be used to show data coming from a Dataset. Users can select a dataset when dragging some of the reporting components to the page in the _Data_ tab and change the reporting component looking in the _Display_ tab.

Reviewer: ```suggestion
You can use the reporting components to display the data coming from a dataset. You can select a dataset by dragging some of the reporting components to the page in the _Data_ tab and change the reporting component from the _Display_ tab.
```

.Reporting Component Editor
image::BAM/ReportingComponentEditor.png[]

== Heatmaps

.Heatmaps Components
image::BAM/HeatmapComponents.png[]

> Heatmaps are components used to display heat information over a process diagram. The heat information can be retrieved from an execution server dataset and used with some of the heatmap components. Any kind of information can be used to build the heatmap including process node execution time and hits.

Reviewer: ```suggestion
Heatmaps components are used to display heat information over a process diagram. You can retrieve the heat information from an execution server dataset which you can use with some of the heatmap components. You can use any kind of information to build the heatmap including process node execution time and hits.
```

.Process Heatmap Component
image::BAM/ProcessHeatmap.png[]

The dataset used with the heatmap is a crucial part to build a meaningful visualization. The *Process Heatmap* requires users to provide two columns: *node id* and *heat value*. In component properties users must provide the *server template*, *container* and *process id*.

> .Process Heatmap Component

Reviewer: ```suggestion
.Processes Heatmap Component
```

image::BAM/ProcessHeatmapDataset.png[]

> The *Processes Heatmaps* is a generic component which dataset must provide *container*, *process* and *node id* with the *heat value*. The only required provide is *server template*. The component allow users to select a process to be displayed

Reviewer 2: ```suggestion
The *Processes Heatmaps* is a generic component which dataset must provide *container*, *process* and *node id* with the *heat value*. The only required property is *server template*. The component allows users to select a process to be displayed
```
The only required provide is -> The only required **property** is
The component allow -> The component **allows**
